jsonnet/kube-thanos/kube-thanos-store.libsonnet

```diff
@@ -120,6 +120,56 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
     },
   },
 
+  withMemcachedIndexCache:: {
+    local ts = self,
+    config+:: {
+      memcached+: {
+        addresses: error 'must provide memcached addresses',
+        timeout: '0s',
```

Comment:
@pracucci  @pstibrany I'm trying to come up with sane default for memcached, could point me to the right direction? What works for you best?

Reply:
It really depends on the scale you wanna address. I still think the defaults I've set into Thanos are OK for a small/medium scale:
https://github.com/thanos-io/thanos/blob/1a2edf19475b854181e5abaa4b6f1c9caf2a1547/pkg/cacheutil/memcached_client.go#L35-L44

If you wanna be a bit more aggressive you may consider:
```
timeout: '2s',
maxIdleConnections: 1000,
maxAsyncConcurrency: 100,
maxAsyncBufferSize: 100000,
maxGetMultiConcurrency: 900,
maxGetMultiBatchSize: 1000,
```

I've also seen you set `dnsProviderUpdateInterval: '3s'`. I'm not sure there's a real value of being that aggressive here. The default 10s could be enough (but maybe you find a good reason to lower it).

Reply:
Thanks a lot for the response, this was really helpful.